content/docs/alerting/configuration.md

```diff
@@ -98,6 +94,12 @@ global:
   # The default HTTP client configuration
   [ http_config: <http_config> ]
 
+  # ResolveTimeout is the default value after which alertmanager
```

Comment:
If a client does not include EndsAt in an alert, default it to this many seconds from now.
This has no impact on alerts from Prometheus, as they always include EndsAt.



2.4.0 was a while back, and I'd prefer to keep this short. I'm only proposing to go beyond pure reference docs here as it's a common confusion.

Reply:
I changed it a bit more to reflect that it has no affect on alerts from Prometheus.

I left `default it to this many seconds from now.` out since this is a duration and can be set in minutes, and also the default value is in minutes too.